pkgs/development/libraries/libeduvpn-common/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchurl
+, gcc
+, pkg-config
```

Comment:
gcc and pkg-config are not needed for me to build and run

Reply:
oh, pkg-config was indeed obvious that it wasn't needed. gcc not so much, I guess go already includes that